### ISSUE TITLE
Always load latin subset?

### ISSUE THREAD

**Issue by davelab6** [Wakamai-Fondue/wakamai-fondue-site#130]
The beta offers Google Fonts split by subset, but when loading an subset other than `latin`, the preview defaults don't work, the family name can't be rendered in itself, etc. So it might be worth loading both latin and the other requested font, so that the latin is available on the page, even though the data isn't about it.

**Comment by RoelN** [comment]
I agree that'd fix the symptom, but would falsely suggest these characters are in the picked font when they're not.

The tester showing tofu instead of text sounds like a feature to me, not a bug :-) At least in the absence of a "fallback text" based on what the font _does_ support.

I have a rough prototype for dropping multiple fonts in the fondue. If priority would be given to the font with the Unicode range of the font's name (Latin in case of Google Fonts' variable fonts), they can show the font name as intended on the first "tab".

Another option would be to have a clear fallback font in place, instead of tofu.

So, options:

1. Fake it by secretly also loading Latin font, fixes things only for Google Fonts
2. If no Latin chars, use fallback font for title, "random" characters that _are_ in the font for tester text
3. Finish multiple font support, putting Latin version on 1st tab, other subsets on the rest

@davelab6 Do any of those options appeal to you?

**Comment by RoelN** [comment]
Closing this if you don't have any objections, and we'll address importing remote fonts in the next version!
